tests/utils/profilehandler/data_preparation.go

```diff
@@ -0,0 +1,222 @@
+package profilehandler
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/tests/utils/common"
+	CON "github.com/openshift/rosa/tests/utils/common/constants"
```

Comment:
NP : It is not common practise in go to capitalize these import aliases. They are typically written in lowercase. 

Not a blocker to this PR, feel free to resolve this comment if you are happy to leave this as is.

Reply:
Updated